src/controllers/v1.controller.ts

```diff
@@ -312,6 +312,26 @@ export class V1Controller {
         throw new ErrorObject(reqRPCID, new jsonrpc.JsonRpcError('No application found in the load balancer', -32055))
       }
 
+      if (!gigastakeOptions.gigastaked) {
+        const { shouldLimit, numRateLimitedApps } = await shouldRateLimit(application.id, this.cache)
+        if (numRateLimitedApps === 0) {
+          logger.log('warn', 'Rate-limited applications list is empty; rate-limiting disabled')
```

Comment:
this warn log would appear on each relay in case the list is empty, which can occur large log ingestion charges from our log management system and is something that, from a coding side, we already know if we left the list empty. I recommend to perform the check of existing apps in rate limit in `applications.ts` and throw the warning there so is made only once

Reply:
Thank you for the review. Fixed.